app/uk/gov/hmrc/merchandiseinbaggage/config/AppConfig.scala

```diff
@@ -73,7 +73,8 @@ trait MibConfiguration {
   lazy val baseUrl: String = "/declare-commercial-goods"
   lazy val declarationsUrl: String = s"$baseUrl/declarations"
   lazy val calculationUrl: String = s"$baseUrl/calculation"
-  lazy val sendEmailsUrl: String = "/declare-commercial-goods/declarations/sendEmails"
+  lazy val sendEmailsUrl: String = s"$declarationsUrl/sendEmails"
+  lazy val checkEoriUrl: String = s"$declarationsUrl/validate/eori/"
```

Comment:
are you sure this is right?

will result in /declare-commercial-goods/declarations/validate/eori/

I think you just want $baseUrl

Reply:
yes! you are right ...good catch